besthouse/main_test.go

```
@@ -0,0 +1,61 @@
+package besthouse
+
+import (
+	"fmt"
+	"testing"
+)
+
+func testGenerateMapValidate(t *testing.T, x, y int) {
+	fmt.Printf("Ask: %d, %d\n", x, y)
+
+	m, err := GenerateMap(x, y)
+
+	// If there is an error...
+	if err != nil {
+		// But for a good reason...
+		if err == ErrInvalidArguments && x < 1 || y < 1 {
+			// We let it pass
+			fmt.Printf("Get: nil\n")
+			return
```

Is this an expected case for success? It probably should return an error and have the caller fail if the error is not present.